src/modules/app/components/Navigator.js

```diff
@@ -74,7 +76,7 @@ export const AppStack = createStackNavigator(
 
 export const LandingStack = createSwitchNavigator(
   {
-    //            The translate() HOC does not calculate props correctly right now. Because the connect
+    // The translate() HOC does not calculate props correctly right now. Because the connect
     // $FlowFixMe in LandingContainer needs the translation HOC there is a flow error here. (NATIVE-53)
```

Comment:
@maxammann any idea what to do about this? Haven't had time yet to fix this (also the two issues above). If not, I'll try to figure it out. But sounds like you had an idea already here.

Reply:
I will look into them but these should be fixed in this PR :)